### ISSUE TITLE
Wallet: accounts page - get real data for all of the NFTs in a users account

### ISSUE THREAD

**Issue by J-Son89** [status-im/status-react#17765]
get real data for all of the NFTs in a users account. align with desktop team before starting this work.

designs:
https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1070-283498&mode=design&t=K0hD3hXfHyjRCciH-4

acceptance criteria:
- all mock data removed and only showing real data
- write tests for new subs & events

**Comment by J-Son89** [comment]
duplicate of https://github.com/status-im/status-mobile/issues/17299
